.github/workflows/welcome.yml

```diff
@@ -12,12 +12,12 @@ jobs:
         with:
           FIRST_ISSUE: |
             👋 @{{ author }}
-             Thanks for opening your first issue here! Be sure to follow the issue template!
+             We appreciate your contribution in UMatter! or something like that?
 
           FIRST_PR: |
             👋 @{{ author }}
             Thanks for opening this pull request! Please check out our contributing guidelines.
 
           FIRST_PR_MERGED: |
             🎉 @{{ author }}
-            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
\ No newline at end of file
+            Congrats on merging your first pull request! We here at behaviorbot are proud of you!
```

Comment:
Replace this-- We here at behaviorbot are proud of you!
to: We appreciate your contribution in UMatter!

Reply:
resolved All @MonalikaPatnaik